diff --git a/node_modules/react-native-calendars/src/calendar-list/index.js b/node_modules/react-native-calendars/src/calendar-list/index.js
index 48fcc35..227a295 100644
--- a/node_modules/react-native-calendars/src/calendar-list/index.js
+++ b/node_modules/react-native-calendars/src/calendar-list/index.js
@@ -287,6 +287,7 @@ class CalendarList extends Component {
         <FlatList
           ref={c => (this.listView = c)}
           style={[this.style.container, style]}
+          onScroll={this.props.onScroll}
           initialListSize={pastScrollRange + futureScrollRange + 1} // ListView deprecated
           data={this.state.rows}
           renderItem={this.renderItem}
